docs/guides/genomics-platform/analyzing-data/sequencerr.md

```diff
@@ -22,22 +22,39 @@ By sequencing a common DNA library on different sequencers, we demonstrate that
 
 ## Outputs
 
-| Name           | Format | Description                                       |
+| Name            | Format | Description                                       |
 | -------------- | ------ | ------------------------------------------------- |
 | PairError file | `.txt` | Base concordance/discordance counts               |
 | Counts file    | `.txt` | Base call frequencies for each genomic coordinate |
 
 
 ## Running the Analysis
 
-Refer to the [analysis workflow guide](../../../files/guides/tools/sequencerr/SequencErr_Cloud_App_Instructions.pdf) to learn how to launch the workflow, hook up input files, adjust parameters, start a run, and monitor run progress.
+Please refer to the following steps to learn how to launch the workflow, hook up input files, adjust parameters, start a run, and monitor run progress.
 
 You can navigate to the **SequencErr** application [here](https://platform.stjude.cloud/workflows/sequencerr).
 
-### Selecting parameters
+!!! note
+    This tool is intended free-of-charge for non-profit usages. 
+    Please contact [Dr. Xiaotu Ma](mailto:Xiaotu.Ma@stjude.org) for for-profit usages and modifications
+    
+### Log in, Run the app and Select the Project
```

Comment:
```suggestion
**Log in, Run the app and Select the Project**
```

Reply:
I would just bold these, making them an H3 makes them show up in the table of contents to the right, which really pollutes that section.